mmdet/models/losses/kd_loss.py

```diff
@@ -0,0 +1,94 @@
+import mmcv
+import torch.nn as nn
+import torch.nn.functional as F
+
+from ..builder import LOSSES
+from .utils import weighted_loss
+
+
+@mmcv.jit(derivate=True, coderize=True)
+@weighted_loss
+def knowledge_distillation_loss(pred, soft_label, T, detach_target=True):
+    r"""Knowledge distillation is a model compression method introduced
+    by Hinto in 2015, which transfer the knowledge from a large model to a
+    small one without loss of validity.
+    <https://arxiv.org/abs/1503.02531>`_.
```

Comment:
Incorrect reference grammar here. You may refer to https://github.com/ralsina/rst-cheatsheet/blob/master/rst-cheatsheet.rst.